src/main/java/uk/gov/hmcts/reform/civil/service/mediation/MediationCsvServiceFactory.java

```diff
@@ -3,17 +3,22 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
 
 @Service
 @RequiredArgsConstructor
 public class MediationCsvServiceFactory {
 
     private final MediationCSVLrvLrService mediationServiceLrvLr;
     private final MediationCSVLrvLipService mediationCSVLrvLipService;
+    private final MediationCSVLipVLipService mediationCSVLipVLipService;
+    private final FeatureToggleService toggleService;
 
     public MediationCSVService getMediationCSVService(CaseData caseData) {
-        if (caseData.isRespondent1LiP()) {
+        if (caseData.isRespondent1LiP() && !caseData.isApplicantLiP()) {
             return mediationCSVLrvLipService;
+        } else if (caseData.isApplicantLiP() && toggleService.isLipVLipEnabled()) {
```

Comment:
can we use isLipvLipOneVOne() instead isApplicantLiP() ??

Reply:
modified to call isLipvLipOneVOne()